### PR TITLE
CI github actions.

### PR DESCRIPTION
**Pull request by Srujan-D**
I have used the most starred repos of the respective formatting software. I'm not sure whether I have used them in the correct way.

### CODE REVIEW

**Comment by threewisemonkeys-as** [comment]
```
      # Install Dependencies
      - name: Install dependencies
        run: |
          python -m pip install --upgrade pip
          pip install flake8 black isort

      - name: Lint check
        run: |
          flake8 .
          black --check .
          isort --profile black -c .
```

Could you try doing something like this instead of using 3 seperate actions? Would be better since we can easily edit which version we want etc...

**Comment by threewisemonkeys-as** [comment]
https://docs.github.com/en/actions/configuring-and-managing-workflows/

You can check out docs for more details

**Comment by Srujan-D** [comment]
> Could you try doing something like this instead of using 3 seperate actions? Would be better since we can easily edit which version we want etc...

Yeah okay, should I make 3 different runs for flake8, isort and black, or do it in 1 run itself? There will be only 1 job in both the cases.


      - name: Lint with flake8
        run: |
          flake8 .
        
      - name: Lint with black
        run: |
          black --check .
          
      -name: Lint with isort
        run: |
          isort --profile black -c .

**Comment by threewisemonkeys-as** [comment]
Yeah, that works too

File: .github/workflows/formatting.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Lint Check
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            python-version: [3.6, 2.7]
```
Comment (threewisemonkeys-as) [inline .github/workflows/formatting.yml]: for linting we only need 3.6

**Review by threewisemonkeys-as — CHANGES_REQUESTED** [review]

**Review by threewisemonkeys-as — APPROVED** [review]

**Pull request merged**